java/code/src/com/suse/manager/reactor/messaging/RegisterMinionEventMessageAction.java

```diff
@@ -1003,98 +1002,6 @@ private Map<String, List<String>> parseRHELReleseQuery(String result) {
     private String getOsRelease(String minionId, ValueMap grains) {
```

Comment:
I think this method is only use in the places you have removed the call. Can this method be removed at all?

Reply:
yes, and there are more unused functions now. I will adapt this